src/panels/lovelace/editor/hui-dialog-migrate-card.ts

```diff
@@ -0,0 +1,83 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
```

Comment:
Should we look into adding a Lovelace/editor Localize for stuff like this? Usually not full sentences in the translation file but this is a pretty descriptive and useful message that english may not be the best hardcoded route? Just a thought

Reply:
Added it.

Reply:
Nice!